test/integration/data-types.test.js

```diff
@@ -460,5 +460,26 @@ describe(Support.getTestDialectTeaser('DataTypes'), function () {
         expect(record.stamp).to.be.eql(testDate);
       });
   });
+  
+  if (dialect === 'mssql') {
+    it('should parse ROWVERSION field', function () {
+      const Model = this.sequelize.define('model', {
+        name: Sequelize.STRING,
+        version: Sequelize.ROWVERSION
+      });
+
+      const sampleData = {
+        name: 'test'
+      };
 
+      return Model.sync({ force: true }).then(() => {
+        return Model.create(sampleData);
+      }).then(() => {
+        return Model.findAll().get(0);
+      }).then(model => {
+        expect(model.get('version')).to.be.ok;
+        expect(typeof model.get('version')).to.be.eql('object');
```

Comment:
Sorry, I see why you understood it as a time-based number. I was going to fix the comments, but I realise the comments for data-types are all gone. This should be the correct comment.

```
/**
 * A rowversion column. Only available in mssql.
 * An incrementing, unique binary number that does not does not preserve a date or a time.
 * @property ROWVERSION
 */
```